.github/workflows/win.yml

```diff
@@ -111,14 +109,26 @@ jobs:
             -DENABLE_CCACHE=OFF \
             -DENABLE_SERVICES=OFF \
             -DENABLE_BENCHMARKS=OFF \
-            -DPREFER_EXTERNAL=ON \
+            -DPREFER_EXTERNAL=ON
+
+      - if: ${{ steps.vcpkg-restore.outputs.cache-hit != 'true' }}
+        name: Save vcpkg packages (if cache miss)
+        uses: actions/cache/save@v3
+        with:
+          key: ${{ steps.vcpkg-restore.outputs.cache-primary-key }}
+          path: |
+            vcpkg/*
+            !vcpkg/downloads
+            !vcpkg/docs
+            !vcpkg/buildtrees
+            vcpkg/downloads/tools
```

Comment:
this saves 500 smth MB (150 or so compressed). keeping all subdirectories of `./vcpkg` easily triples the size.